Frends.MySQL.ExecuteProcedure/Frends.MySQL.ExecuteProcedure/Definitions/Input.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.MySQL.ExecuteProcedure.Definitions;
+
+/// <summary>
+/// Input class
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Mysql connection string
+    /// </summary>
+    /// <example>server=server;user=user;database=db;password=pw;</example>
+    [PasswordPropertyText]
+    [DisplayFormat(DataFormatString = "Text")]
```

Comment:
You don't need this line since `PasswordPropertyText` will set the default input type to Text.